lib/server/worker.js

```diff
@@ -52,6 +52,16 @@ exports.create = function() {
     }
   });
 
+  server.route({
+    method: 'DELETE',
+    path: '/a/{id}',
+    config: {
+      handler: function delete(req, reply) {
```

Comment:
`grunt lint` says:

> _Expected an identifier and instead saw 'delete' (a reserved word)._

Not sure if that matters (unless you're IE8 or something), but we may want to relax that JSHint warning. Plus, I/we should look into whether there the `grunt lint` task is, in fact, being run on Travis.